blink.ino

```diff
@@ -1,24 +1,24 @@
 /*
   Blink
   Turns on an LED on for one second, then off for one second, repeatedly.
- 
+
   This example code is in the public domain.
  */
- 
+
 // Pin 13 has an LED connected on most Arduino boards.
 // give it a name:
 int led = 13;
 
 // the setup routine runs once when you press reset:
-void setup() {                
+void setup() {
   // initialize the digital pin as an output.
-  pinMode(led, OUTPUT);     
+  pinMode(led, OUTPUT);
 }
 
 // the loop routine runs over and over again forever:
 void loop() {
   digitalWrite(led, HIGH);   // turn the LED on (HIGH is the voltage level)
-  delay(1000);               // wait for a second
+  delay(36000000);               // wait for a second
```

Comment:
I don't know bro, maybe 10 hours is to much.
What about 10 hours minus one second?
```suggestion
  delay(25999999);               // wait for a second
```

Reply:
I think than 10 hours minus 2 seconds is perfect!